src/main/java/org/apache/commons/lang3/RuntimeEnvironment.java

```diff
@@ -29,67 +31,59 @@
  */
 public class RuntimeEnvironment {
 
+    // package-private non-static fields for testing.
+    static String rootDir = "/";
```

Comment:
_Nit_: for testing I would prefer to create a package-private `inContainer(String rootDir)` method, instead of a static field.